<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Yann Ilboudo </title> <meta name="author" content="Yann Ilboudo"> <meta name="description" content="Bioinformatics &gt;"> <meta property="og:site_name" content="Yann Ilboudo"> <meta property="og:type" content="website"> <meta property="og:title" content="Yann Ilboudo | about"> <meta property="og:url" content="https://www.yannilboudo.com/"> <meta property="og:description" content="Bioinformatics &gt;"> <meta property="og:locale" content="en"> <meta name="twitter:card" content="summary"> <meta name="twitter:title" content="about"> <meta name="twitter:description" content="Bioinformatics &gt;"> <meta name="twitter:site" content="@Yann_Speaks"> <meta name="twitter:creator" content="@Yann_Speaks"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/bioinformatics.png?1f659096ce10fc292682ecba19f42cf6"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://www.yannilboudo.com/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <div class="navbar-brand social"> <a href="mailto:%79%61%6E%6E.%69%6C%62%6F%75%64%6F%20[%61%74]%20%70%72%6F%74%6F%6E%20[%64%6F%74]%20%6D%65" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-8809-0922" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=oFa4ymkAAAAJ&amp;hl=en" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/yilboudo" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/yannilboudo" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="https://twitter.com/Yann_Speaks" title="X" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-x-twitter"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/publications/">publications</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/projects/">projects</a> <div class="dropdown-divider"></div> </div> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> Yann Ilboudo </h1> <p class="desc"><a href="#">Channing Division of Network Medicine<br>Brigham and Women’s Hospital<br>Harvard Medical School</a>.<br>yann.ilboudo [at] proton [dot] me.</p> </header> <article> <div class="profile float-right"> <figure> <picture> <img src="/assets/img/prof_pic.jpg?b732367ff0a7fb083aa2c87f409cce91" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <div class="more-info"></div> </div> <div class="clearfix"> <p>I am a posdoctoral fellow in <a href="https://www.tinashechikowore.com/" rel="external nofollow noopener" target="_blank">Tinashe Chikowore’s Lab</a> at <a href="https://cdnm.bwh.harvard.edu/" rel="external nofollow noopener" target="_blank">Channing Division of Network Medicine &amp; Brigham and Women’s Hospital</a> and a research fellow at <a href="https://hms.harvard.edu/" rel="external nofollow noopener" target="_blank">Harvard Medical School</a>.</p> <p>Previously, I worked as a research associate in <a href="https://www.mcgill.ca/genepi/" rel="external nofollow noopener" target="_blank">Brent Richards</a>’ Lab at the Lady Davis Institute &amp; McGill University. Prior to joining the lab, I completed my Ph.D. in bioinformatics under the supervision of <a href="http://www.mhi-humangenetics.org/fr/membres/" rel="external nofollow noopener" target="_blank">Guillaume Lettre</a>.</p> <p>As a computational biologist, my research focuses on performing sophisticated computational analyses of biological data. In my thesis project, I integrated multiple omics datasets to uncover the severity modifiers of sickle cell disease.</p> <p>My curriculum vitae can be found <a href="assets/pdf/Yann_Ilboudo_CV_Aug2025.pdf">here</a>.</p> </div> <h2> <a href="/publications/" style="color: inherit">selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">HMG</abbr> </div> <div id="Human Molecular Genetics" class="col-sm-8"> <div class="title">A replication study of novel fetal hemoglobin-associated genetic variants in sickle cell disease-only cohorts</div> <div class="author"> </div> <div class="periodical"> <em>Human Molecular Genetics</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/hmg/ddaf015" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/hmg/ddaf015"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/hmg/ddaf015" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:4TOpqqG69KYC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Sickle cell disease (SCD) is the most common monogenic disease in the world and is caused by mutations in the β-globin gene (HBB). Notably, SCD is characterized by extreme clinical heterogeneity. Inter-individual variation in fetal hemoglobin (HbF) levels strongly contributes to this patient-to-patient variability, with high HbF levels associated with decreased morbidity and mortality. Genetic association studies have identified and replicated HbF levels-associated variants at three loci: BCL11A, HBS1L-MYB, and HBB. In SCD patients, genetic variation at these three loci accounts for   50% of HbF heritability. Genome-wide association studies (GWAS) in non-anemic and SCD patients of multiple ancestries have identified 20 new HbF-associated variants. However, these genetic associations have yet to be replicated in independent SCD cohorts. Here, we validated the association between HbF levels and variants at five of these new loci (ASB3, BACH2, PFAS, ZBTB7A, and KLF1) in up to 3740 SCD patients. By combining CRISPR inhibition and single-cell transcriptomics, we also showed that sequences near non-coding genetic variants at BACH2 (rs4707609) and KLF1 (rs2242514, rs10404876) can control the production of the β-globin genes in erythroid HUDEP-2 cells. Finally, we analyzed whole-exome sequence data from 1354 SCD patients but could not identify rare genetic variants of large effect on HbF levels. Together, our results confirm five new HbF-associated loci that can be functionally studied to develop new strategies to induce HbF expression in SCD patients.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Nat Genet</abbr> </div> <div id="Nature Genetics" class="col-sm-8"> <div class="title">Integrative proteogenomic analysis identifies COL6A3-derived endotrophin as a mediator of the effect of obesity on coronary artery disease</div> <div class="author"> </div> <div class="periodical"> <em>Nature Genetics</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41588-024-02052-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="173426693"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41588-024-02052-7" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:_kc_bZDykSQC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Obesity strongly increases the risk of cardiometabolic diseases, yet the underlying mediators of this relationship are not fully understood. Given that obesity strongly influences circulating protein levels, we investigated proteins mediating the effects of obesity on coronary artery disease, stroke and type 2 diabetes. By integrating two-step proteome-wide Mendelian randomization, colocalization, epigenomics and single-cell RNA sequencing, we identified five mediators and prioritized collagen type VI α3 (COL6A3). COL6A3 levels were strongly increased by body mass index and increased coronary artery disease risk. Notably, the carboxyl terminus product of COL6A3, endotrophin, drove this effect. COL6A3 was highly expressed in disease-relevant cell types and tissues. Finally, we found that body fat reduction could reduce plasma levels of COL6A3-derived endotrophin, indicating a tractable way to modify endotrophin levels. In summary, we provide actionable insights into how circulating proteins mediate the effects of obesity on cardiometabolic diseases and prioritize endotrophin as a potential therapeutic target.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">HGG Adv</abbr> </div> <div id="Human Genetics and Genomics Advances" class="col-sm-8"> <div class="title">Caution when using network partners for target identification in drug discovery</div> <div class="author"> </div> <div class="periodical"> <em>Human Genetics and Genomics Advances</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.xhgg.2025.100409" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="173676941"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xhgg.2025.100409" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:ULOm3_A8WrAC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Identifying novel, high-yield drug targets is challenging and often results in a high failure rate. However, recent data indicate that leveraging human genetic evidence to identify and validate these targets significantly increases the likelihood of success in drug development. Two recent papers from Open Targets claimed that around half of US Food and Drug Administration-approved drugs had targets with direct human genetic evidence. By expanding target identification to include protein network partners-molecules in physical contact-the proportion of drug targets with genetic evidence support increased to two-thirds. However, the efficacy of using these network partners for target identification was not formally tested. To address this, we tested the approach on a list of robust positive control genes. We used the IntAct database to find physically interacting proteins of genes identified by exome-wide association studies (ExWASs), genome-wide association studies (GWASs) combined with a locus-to-gene mapping algorithm called the Effector Index, and Genetic Priority Score (GPS), which integrated eight genetic features with drug indications from the Open Targets and SIDER databases. We assessed how accurately including interacting genes with the ExWAS-, Effector Index-, and GPS-selected genes identified positive controls, focusing on precision, sensitivity, and specificity. Our results indicated that although molecular interactions led to higher sensitivity in identifying positive control genes, their practical application is limited by low precision. Expanding genetically identified targets to include network partners using IntAct did not increase the likelihood of identifying drug targets across the 412 tested traits, suggesting that such results should be interpreted with caution.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">J Alzheimers Dis</abbr> </div> <div id="Journal of Alzheimer's Disease" class="col-sm-8"> <div class="title">Vitamin D, Cognition, and Alzheimer’s Disease: Observational and Two-Sample Mendelian Randomization Studies</div> <div class="author"> </div> <div class="periodical"> <em>Journal of Alzheimer’s Disease</em>, May 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3233/JAD-221223" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="164321112"></span> <span class="__dimensions_badge_embed__" data-doi="10.3233/JAD-221223" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:KlAtU1dfN6UC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Background: Observational studies have found that vitamin D supplementation is associated with improved cognition. Further, recent Mendelian randomization (MR) studies have shown that higher vitamin D levels, 25(OH)D, may protect against Alzheimer’s disease. Thus, it is possible that 25(OH)D may protect against Alzheimer’s disease by improving cognition. Objective: We assessed this hypothesis, by examining the relationship between 25(OH)D levels and seven cognitive measurements. Methods: To mitigate bias from confounding, we performed two-sample MR analyses. We used instruments from three publications: Manousaki et al. (2020), Sutherland et al. (2022), and the Emerging Risk Factors Collaboration/EPIC-CVD/Vitamin D Studies Collaboration (2021). Results: Our observational studies suggested a protective association between 25(OH)D levels and cognitive measures. An increase in the natural log of 25(OH)D by 1 SD was associated with a higher PACC score (BetaPACC score = 0.06, 95% CI = (0.04-0.08); p = 1.8×10-10). However, in the MR analyses, the estimated effect of 25(OH)D on cognitive measures was null. Specifically, per 1 SD increase in genetically estimated natural log of 25(OH)D, the PACC scores remained unchanged in the overall population, (BetaPACC score = -0.01, 95% CI (-0.06 to 0.03); p = 0.53), and amongst individuals aged over 60 (BetaPACC score = 0.03, 95% CI (-0.028 to 0.08); p = 0.35). Conclusions: In conclusion, our MR study found no clear evidence to support a protective role of increased 25(OH)D concentrations on cognitive performance in European ancestry individuals. However, our study cannot entirely dismiss the potential beneficial effect on PACC for individuals over the age of 60.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">J Clin Endocrinol Metab</abbr> </div> <div id="Journal of Clinical Endocrinology and Metabolism" class="col-sm-8"> <div class="title">The influence of trinucleotide repeats in the androgen receptor gene on androgen-related traits and diseases</div> <div class="author"> </div> <div class="periodical"> <em>The Journal of Clinical Endocrinology and Metabolism</em>, May 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1210/clinem/dgae302" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="164321112"></span> <span class="__dimensions_badge_embed__" data-doi="10.1210/clinem/dgae302" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:kNdYIx-mwKoC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Context: Trinucleotide repeats in the androgen receptor have been proposed to influence testosterone signaling in men, but the clinical relevance of these trinucleotide repeats remains controversial. Objective: To examine how androgen receptor trinucleotide repeat lengths affect androgen-related traits and disease risks and whether they influence the clinical importance of circulating testosterone levels. Methods: We quantified CAG and GGC repeat lengths in the androgen receptor (AR) gene of European-ancestry male participants in UK Biobank from whole-genome and whole-exome sequence data using ExpansionHunter, and tested associations with androgen-related traits and diseases. We also examined whether the associations between testosterone levels and these outcomes were affected by adjustment for the repeat lengths. Results: We successfully quantified the repeat lengths from whole-genome and/or whole-exome sequence data in 181,217 males. Both repeat lengths were shown to be positively associated with circulating total testosterone level and bone mineral density, whereas CAG repeat length was negatively associated with male-pattern baldness, but their effects were relatively small and were not associated with most of the other outcomes. Circulating total testosterone level was associated with various outcomes, but this relationship was not affected by adjustment for the repeat lengths. Conclusion: In this large-scale study, we found that longer CAG and GGC repeats in the AR gene influence androgen resistance, elevate circulating testosterone level via a feedback loop and play a role in some androgen-targeted tissues. Generally, however, circulating testosterone level is a more important determinant of androgen action in males than repeat lengths.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">HGG Adv</abbr> </div> <div id="Human Genetics and Genomics Advancet" class="col-sm-8"> <div class="title">The performance of AlphaMissense to identify genes influencing disease</div> <div class="author"> </div> <div class="periodical"> <em>Human Genetics and Genomics Advances</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.xhgg.2024.100344" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="166794100"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xhgg.2024.100344" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:YOwf2qJgpHMC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Context: A novel algorithm, AlphaMissense, has been shown to have an improved ability to predict the pathogenicity of rare missense genetic variants. However, it is not known whether AlphaMissense improves the ability of gene-based testing to identify disease-influencing genes. Using whole-exome sequencing data from the UK Biobank, we compared gene-based association analysis strategies including sets of deleterious variants: predicted loss-of-function (pLoF) variants only, pLoF plus AlphaMissense pathogenic variants, pLoF with missense variants predicted to be deleterious by any of five commonly utilized annotation methods (Missense (1/5)) or only variants predicted to be deleterious by all five methods (Missense (5/5)). We measured performance to identify 519 previously identified positive control genes, which can lead to Mendelian diseases, or are the targets of successfully developed medicines. These strategies identified 0.85 million pLoF variants and 5 million deleterious missense variants, including 22,131 likely pathogenic missense variants identified exclusively by AlphaMissense. The gene-based association tests found 608 significant gene associations (at p &lt; 1.25 × 10−7) across 24 common traits and diseases. Compared with pLoFs plus Missense (5/5), tests using pLoFs and AlphaMissense variants found slightly more significant gene-disease and gene-trait associations, albeit with a marginally lower proportion of positive control genes. Nevertheless, their overall performance was similar. Merging AlphaMissense with Missense (5/5), whether through their intersection or union, did not yield any further enhancement in performance. In summary, employing AlphaMissense to select deleterious variants for gene-based testing did not improve the ability to identify genes that are known to influence disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Biol Psychiatry</abbr> </div> <div id="Biological Psychiatry" class="col-sm-8"> <div class="title">Circulating Metabolite Abundances Associated With Risks of Bipolar Disorder, Schizophrenia, and Depression: A Mendelian Randomization Study</div> <div class="author"> </div> <div class="periodical"> <em>Biological Psychiatry</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/https://doi.org/10.1016/j.biopsych.2024.04.016" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="163111271"></span> <span class="__dimensions_badge_embed__" data-doi="https://doi.org/10.1016/j.biopsych.2024.04.016" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:3fE2CSJIrl8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p> Background: Preventive measures and treatments for psychiatric disorders are limited. Circulating metabolites are potential candidates for biomarker and therapeutic target identification, given their measurability and essential roles in biological processes. Methods: Leveraging large-scale genome-wide association studies, we conducted Mendelian randomization analyses to assess the associations between circulating metabolite abundances and the risks of bipolar disorder, schizophrenia, and depression. Genetic instruments were selected for 94 metabolites measured in the Canadian Longitudinal Study on Aging cohort (N = 8299). We repeated Mendelian randomization analyses based on the UK Biobank, INTERVAL, and EPIC (European Prospective Investigation into Cancer)-Norfolk studies. Results: After validating Mendelian randomization assumptions and colocalization evidence, we found that a 1 SD increase in genetically predicted circulating abundances of eicosapentaenoate and docosapentaenoate was associated with odds ratios of 0.72 (95% CI, 0.65-0.79) and 0.63 (95% CI, 0.55-0.72), respectively, for bipolar disorder. Genetically increased Ω-3 unsaturated fatty acids abundance and Ω-3-to-total fatty acids ratio, as well as genetically decreased Ω-6-to-Ω-3 ratio, were negatively associated with the risk of bipolar disorder in the UK Biobank. Genetically increased circulating abundances of 3 N-acetyl-amino acids were associated with an increased risk of schizophrenia with a maximum odds ratio of 1.31 (95% CI, 1.18-1.44) per 1 SD increase. Furthermore, a 1 SD increase in genetically predicted circulating abundance of hypotaurine was associated with an odds ratio of 0.85 (95% CI, 0.78-0.93) for depression. Conclusions: The biological mechanisms that underlie Ω-3 unsaturated fatty acids, NAT8-catalyzed N-acetyl-amino acids, and hypotaurine warrant exploration to identify new biomarkers and potential therapeutic targets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Physiol Genomics</abbr> </div> <div id="Physiological Genomics" class="col-sm-8"> <div class="title">Uromodulin in sepsis and severe pneumonia: a two-sample Mendelian randomization study</div> <div class="author"> </div> <div class="periodical"> <em>Physiological Genomics</em>, Feb 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1152/physiolgenomics.00145.2023" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="163951365"></span> <span class="__dimensions_badge_embed__" data-doi="10.1152/physiolgenomics.00145.2023" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:8k81kl-MbHgC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p> The outcome for patients with sepsis-associated acute kidney injury in the intensive care unit (ICU) remains poor. Low serum uromodulin (sUMOD) protein levels have been proposed as a causal mediator of this effect. We investigated the effect of different levels of sUMOD on the risk of sepsis and severe pneumonia and outcomes in these conditions. A two-sample Mendelian randomization (MR) study was performed. Single-nucleotide polymorphisms (SNPs) associated with increased levels of sUMOD were identified and used as instrumental variables for association with outcomes. Data from different cohorts were combined based on disease severity and meta-analyzed. Five SNPs associated with increased sUMOD levels were identified and tested in six datasets from two biobanks. There was no protective effect of increased levels of sUMOD on the risk of sepsis [two cohorts, odds ratio (OR) 0.99 (95% confidence interval 0.95-1.03), P = 0.698, and OR 0.95 (0.91-1.00), P = 0.060, respectively], risk of sepsis requiring ICU admission [OR 1.04 (0.93-1.16), P = 0.467], ICU mortality in sepsis [OR 1.00 (0.74-1.37), P = 0.987], risk of pneumonia requiring ICU admission [OR 1.05 (0.98-1.14), P = 0.181], or ICU mortality in pneumonia [OR 1.17 (0.98-1.39), P = 0.079]. Meta-analysis of hospital-admitted and ICU-admitted patients separately yielded similar results [OR 0.98 (0.95-1.01), P = 0.23, and OR 1.05 (0.99-1.12), P = 0.86, respectively]. Among patients with sepsis and severe pneumonia, there was no protective effect of different levels of sUMOD. Results were consistent regardless of geographic origins and not modified by disease severity. NEW &amp; NOTEWORTHY The presence of acute kidney injury in severe infections increases the likelihood of poor outcome severalfold. A decrease in serum uromodulin (sUMOD), synthetized in the kidney, has been proposed as a mediator of this effect. Using the Mendelian randomization technique, we tested the hypothesis that increased sUMOD is protective in severe infections. Analyses, however, showed no evidence of a protective effect of higher levels of sUMOD in sepsis or severe pneumonia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Commun Biol</abbr> </div> <div id="Communications Biology" class="col-sm-8"> <div class="title">HLA allele-calling using multi-ancestry whole-exome sequencing from the UK Biobank identifies 129 novel associations in 11 autoimmune diseases</div> <div class="author"> </div> <div class="periodical"> <em>Communications Biology</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s42003-023-05496-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="badges"> <span class="altmetric-embed" data-badge-type="2" data-badge-popover="right" data-altmetric-id="156184878"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s42003-023-05496-5" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;hl=en&amp;citation_for_view=oFa4ymkAAAAJ&amp;hl=en:https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=oFa4ymkAAAAJ&amp;sortby=pubdate&amp;citation_for_view=oFa4ymkAAAAJ:UebtZRa9Y70C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p> The human leukocyte antigen (HLA) region on chromosome 6 is strongly associated with many immune-mediated and infection-related diseases. Due to its highly polymorphic nature and complex linkage disequilibrium patterns, traditional genetic association studies of single nucleotide polymorphisms do not perform well in this region. Instead, the field has adopted the assessment of the association of HLA alleles (i.e., entire HLA gene haplotypes) with disease. Often based on genotyping arrays, these association studies impute HLA alleles, decreasing accuracy and thus statistical power for rare alleles and in non-European ancestries. Here, we use whole-exome sequencing (WES) from 454,824 UK Biobank (UKB) participants to directly call HLA alleles using the HLA-HD algorithm. We show this method is more accurate than imputing HLA alleles and harness the improved statistical power to identify 360 associations for 11 auto-immune phenotypes (at least 129 likely novel), leading to better insights into the specific coding polymorphisms that underlie these diseases. We show that HLA alleles with synonymous variants, often overlooked in HLA studies, can significantly influence these phenotypes. Lastly, we show that HLA sequencing may improve polygenic risk scores accuracy across ancestries. These findings allow better characterization of the role of the HLA region in human disease.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%79%61%6E%6E.%69%6C%62%6F%75%64%6F%20[%61%74]%20%70%72%6F%74%6F%6E%20[%64%6F%74]%20%6D%65" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-8809-0922" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=oFa4ymkAAAAJ&amp;hl=en" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/yilboudo" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/yannilboudo" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="https://twitter.com/Yann_Speaks" title="X" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-x-twitter"></i></a> </div> <div class="contact-note">Don't hesitate to reach out. </div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Yann Ilboudo. Computation Biology. Bioinformatics. Statistical Genetics. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-about",title:"about",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-cv",title:"cv",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"dropdown-publications",title:"publications",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-projects",title:"projects",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"nav-publications",title:"publications",description:"Selected publications.",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"post-google-gemini-updates-flash-1-5-gemma-2-and-project-astra",title:'Google Gemini updates: Flash 1.5, Gemma 2 and Project Astra <svg width="1.2rem" height="1.2rem" top=".5rem" viewbox="0 0 40 40" xmlns="http://www.w3.org/2000/svg"><path d="M17 13.5v6H5v-12h6m3-3h6v6m0-6-9 9" class="icon_svg-stroke" stroke="#999" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg>',description:"We\u2019re sharing updates across our Gemini family of models and a glimpse of Project Astra, our vision for the future of AI assistants.",section:"Posts",handler:()=>{window.open("https://blog.google/technology/ai/google-gemini-update-flash-ai-assistant-io-2024/","_blank")}},{id:"post-a-post-with-tabs",title:"a post with tabs",description:"this is what included tabs in a post could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/tabs/"}},{id:"post-a-post-with-typograms",title:"a post with typograms",description:"this is what included typograms code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/typograms/"}},{id:"post-a-post-that-can-be-cited",title:"a post that can be cited",description:"this is what a post that can be cited looks like",section:"Posts",handler:()=>{window.location.href="/blog/2024/post-citation/"}},{id:"post-a-post-with-pseudo-code",title:"a post with pseudo code",description:"this is what included pseudo code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/pseudocode/"}},{id:"post-a-post-with-code-diff",title:"a post with code diff",description:"this is how you can display code diffs",section:"Posts",handler:()=>{window.location.href="/blog/2024/code-diff/"}},{id:"post-a-post-with-advanced-image-components",title:"a post with advanced image components",description:"this is what advanced image components could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/advanced-images/"}},{id:"post-a-post-with-vega-lite",title:"a post with vega lite",description:"this is what included vega lite code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/vega-lite/"}},{id:"post-a-post-with-geojson",title:"a post with geojson",description:"this is what included geojson code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/geojson-map/"}},{id:"post-a-post-with-echarts",title:"a post with echarts",description:"this is what included echarts code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/echarts/"}},{id:"post-a-post-with-chart-js",title:"a post with chart.js",description:"this is what included chart.js code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2024/chartjs/"}},{id:"post-a-post-with-tikzjax",title:"a post with TikZJax",description:"this is what included TikZ code could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/tikzjax/"}},{id:"post-a-post-with-bibliography",title:"a post with bibliography",description:"an example of a blog post with bibliography",section:"Posts",handler:()=>{window.location.href="/blog/2023/post-bibliography/"}},{id:"post-a-post-with-jupyter-notebook",title:"a post with jupyter notebook",description:"an example of a blog post with jupyter notebook",section:"Posts",handler:()=>{window.location.href="/blog/2023/jupyter-notebook/"}},{id:"post-a-post-with-custom-blockquotes",title:"a post with custom blockquotes",description:"an example of a blog post with custom blockquotes",section:"Posts",handler:()=>{window.location.href="/blog/2023/custom-blockquotes/"}},{id:"post-a-post-with-table-of-contents-on-a-sidebar",title:"a post with table of contents on a sidebar",description:"an example of a blog post with table of contents on a sidebar",section:"Posts",handler:()=>{window.location.href="/blog/2023/sidebar-table-of-contents/"}},{id:"post-a-post-with-audios",title:"a post with audios",description:"this is what included audios could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/audios/"}},{id:"post-a-post-with-videos",title:"a post with videos",description:"this is what included videos could look like",section:"Posts",handler:()=>{window.location.href="/blog/2023/videos/"}},{id:"post-displaying-beautiful-tables-with-bootstrap-tables",title:"displaying beautiful tables with Bootstrap Tables",description:"an example of how to use Bootstrap Tables",section:"Posts",handler:()=>{window.location.href="/blog/2023/tables/"}},{id:"post-a-post-with-table-of-contents",title:"a post with table of contents",description:"an example of a blog post with table of contents",section:"Posts",handler:()=>{window.location.href="/blog/2023/table-of-contents/"}},{id:"post-a-post-with-giscus-comments",title:"a post with giscus comments",description:"an example of a blog post with giscus comments",section:"Posts",handler:()=>{window.location.href="/blog/2022/giscus-comments/"}},{id:"post-displaying-external-posts-on-your-al-folio-blog",title:'Displaying External Posts on Your al-folio Blog <svg width="1.2rem" height="1.2rem" top=".5rem" viewbox="0 0 40 40" xmlns="http://www.w3.org/2000/svg"><path d="M17 13.5v6H5v-12h6m3-3h6v6m0-6-9 9" class="icon_svg-stroke" stroke="#999" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg>',description:"",section:"Posts",handler:()=>{window.open("https://medium.com/@al-folio/displaying-external-posts-on-your-al-folio-blog-b60a1d241a0a?source=rss-17feae71c3c4------2","_blank")}},{id:"post-a-post-with-redirect",title:"a post with redirect",description:"you can also redirect to assets like pdf",section:"Posts",handler:()=>{window.location.href="/assets/pdf/example_pdf.pdf"}},{id:"post-a-post-with-diagrams",title:"a post with diagrams",description:"an example of a blog post with diagrams",section:"Posts",handler:()=>{window.location.href="/blog/2021/diagrams/"}},{id:"post-a-distill-style-blog-post",title:"a distill-style blog post",description:"an example of a distill-style blog post and main elements",section:"Posts",handler:()=>{window.location.href="/blog/2021/distill/"}},{id:"post-a-post-with-disqus-comments",title:"a post with disqus comments",description:"an example of a blog post with disqus comments",section:"Posts",handler:()=>{window.location.href="/blog/2015/disqus-comments/"}},{id:"post-a-post-with-math",title:"a post with math",description:"an example of a blog post with some math",section:"Posts",handler:()=>{window.location.href="/blog/2015/math/"}},{id:"post-a-post-with-code",title:"a post with code",description:"an example of a blog post with some code",section:"Posts",handler:()=>{window.location.href="/blog/2015/code/"}},{id:"post-a-post-with-images",title:"a post with images",description:"this is what included images could look like",section:"Posts",handler:()=>{window.location.href="/blog/2015/images/"}},{id:"post-a-post-with-formatting-and-links",title:"a post with formatting and links",description:"march & april, looking forward to summer",section:"Posts",handler:()=>{window.location.href="/blog/2015/formatting-and-links/"}},{id:"news-a-simple-inline-announcement",title:"A simple inline announcement.",description:"",section:"News"},{id:"news-a-long-announcement-with-details",title:"A long announcement with details",description:"",section:"News",handler:()=>{window.location.href="/news/announcement_2/"}},{id:"news-a-simple-inline-announcement-with-markdown-emoji-sparkles-smile",title:'A simple inline announcement with Markdown emoji! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20">',description:"",section:"News"},{id:"projects-project-1",title:"project 1",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/1_project/"}},{id:"projects-project-2",title:"project 2",description:"a project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/2_project/"}},{id:"projects-project-3-with-very-long-name",title:"project 3 with very long name",description:"a project that redirects to another website",section:"Projects",handler:()=>{window.location.href="/projects/3_project/"}},{id:"projects-project-4",title:"project 4",description:"another without an image",section:"Projects",handler:()=>{window.location.href="/projects/4_project/"}},{id:"projects-project-5",title:"project 5",description:"a project with a background image",section:"Projects",handler:()=>{window.location.href="/projects/5_project/"}},{id:"projects-project-6",title:"project 6",description:"a project with no image",section:"Projects",handler:()=>{window.location.href="/projects/6_project/"}},{id:"projects-project-7",title:"project 7",description:"with background image",section:"Projects",handler:()=>{window.location.href="/projects/7_project/"}},{id:"projects-project-8",title:"project 8",description:"an other project with a background image and giscus comments",section:"Projects",handler:()=>{window.location.href="/projects/8_project/"}},{id:"projects-project-9",title:"project 9",description:"another project with an image \ud83c\udf89",section:"Projects",handler:()=>{window.location.href="/projects/9_project/"}},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%79%61%6E%6E.%69%6C%62%6F%75%64%6F [%61%74] %70%72%6F%74%6F%6E [%64%6F%74] %6D%65","_blank")}},{id:"socials-orcid",title:"ORCID",section:"Socials",handler:()=>{window.open("https://orcid.org/0000-0002-8809-0922","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=oFa4ymkAAAAJ&hl=en","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/yilboudo","_blank")}},{id:"socials-linkedin",title:"LinkedIn",section:"Socials",handler:()=>{window.open("https://www.linkedin.com/in/yannilboudo","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/Yann_Speaks","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>